src/app/routes/connector-ui/connector-ui.component.html

```diff
@@ -19,6 +19,7 @@
         <mat-list-item
           *ngIf="
             route.component &&
+            !route.data?.isNotShownInNavBar &&
```

Comment:
i also added sth like that , but i filitered in the component class on the assignment (thinks you would not need in react zzz)